Project/Src/Gui/TextAreaControl.cs

```diff
@@ -46,6 +45,7 @@ public TextAreaControl(TextEditorControl motherTextEditorControl)
             Controls.Add(TextArea);
 
             VScrollBar.ValueChanged += VScrollBarValueChanged;
+            VScrollBar.MouseScrollEnded += VScrollBarOnMouseScrollEnded;
```

Comment:
```suggestion
            VScrollBar.MouseScrollEnded += (_, _) => UpdateLayout();
```

Reply:
While this is simpler it doesn't follow the style of every other event registration, I'd like to leave this as is for consistency.